nntoolkit/create.py

```diff
@@ -43,12 +96,7 @@ def main(nn_type, architecture, model_file):
        :param model_file: A path which should end with .tar. The created model
        will be written there.
     """
-    if os.path.isfile(model_file):
-        logging.error("'%s' already exists.", model_file)
-        return
-    if not model_file.endswith(".tar"):
-        logging.error("'%s' does not end with '.tar'.", model_file)
-        return
+    file_validate(model_file)
```

Comment:
This is not equivalent. The execution of the script should stop if the file is not valid.